examples/go2_fulldynamics.py

```diff
@@ -140,6 +143,9 @@
 
 qp = IDSolver(id_conf, model_handler.getModel())
 
+""" Friction """
+fcompensation = FrictionCompensation(model_handler.getModel(), nu)
```

Comment:
I think you forgot to update this constructor call (nu is passed where `with_free_flyer` should be)